cmd/spire-agent/cli/run/run.go

```diff
@@ -224,11 +226,14 @@ func newAgentConfig(c *config) (*agent.Config, error) {
 	ac.TrustDomain = *td
 
 	// Parse trust bundle
-	bundle, err := parseTrustBundle(c.Agent.TrustBundlePath)
-	if err != nil {
-		return nil, fmt.Errorf("could not parse trust bundle: %s", err)
+	ac.InsecureBootstrap = c.Agent.InsecureBootstrap
```

Comment:
nit: A small test case covering this line would be good

Reply:
good call :)